pyproject.toml

```diff
@@ -102,12 +102,10 @@ testpaths = [
 filterwarnings = [
     "error",
     "ignore::ResourceWarning",
-    # https://github.com/lancedb/lancedb/issues/1296
-    "ignore:Function 'semver.parse_version_info' is deprecated:DeprecationWarning",
-    # chromadb -> opentelemetry -> google
-    "ignore:Type google._upb._message:DeprecationWarning",
-    # chromadb
-    "ignore:Python 3.14 will:DeprecationWarning"
+    # https://github.com/chroma-core/chroma/issues/4190
+    "ignore:Accessing the 'model_fields':pydantic.warnings.PydanticDeprecatedSince211",
+    # https://github.com/lancedb/lancedb/issues/2315
+    "ignore:Support for class-based `config`:pydantic.warnings.PydanticDeprecatedSince20",
```

Comment:
I have PRs on both Chroma and Lance to fix this so hopefully we can remove these soon as well.